pkg/u8proto/u8proto.go

```diff
@@ -41,6 +41,7 @@ var protoNames = map[U8proto]string{
 }
 
 var ProtoIDs = map[string]U8proto{
+	"":       0,
```

Comment:
Why this change?

Reply:
Not sure any more... To map a missing proto name to ANY, but then in K8s ContainerPort's Protocol defaults to "TCP". I'll check if this is still needed & verify the default behavior.

Reply:
It seems my assumption was that ContainerPort could have ANY protocol, but that is not the case, even though this PR would allow it. Our policy rules allow ANY protocol, which we internally treat as both UDP and TCP and generate two rules (one for each). One of them gets pruned away when the port name is resolved.

It might be better if I make this implementation less permissive and only allow one protocol, missing defaulting to TCP?

Reply:
Changed empty proto string to default to TCP, but kept the internal processing that allows "ANY".